Buildings/Experimental/OpenBuildingControl/CDL/Logical/TrueFalseHold.mo

```diff
@@ -0,0 +1,158 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.Logical;
+block TrueFalseHold "Block that holds an output signal for a specified duration"
+
+  parameter Modelica.SIunits.Time holdDuration
+    "Time duration during which the output cannot change";
+
+  Interfaces.BooleanInput u "Boolean input signal"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+        iconTransformation(extent={{-122,-10},{-102,10}})));
+
+  Interfaces.BooleanOutput y "Boolean output signal"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}}),
+        iconTransformation(extent={{100,-10},{120,10}})));
+
+protected
+  Timer timer1 "Timer for false output"
+    annotation (Placement(transformation(extent={{-80,-40},{-60,-20}})));
+
+  Timer timer2 "Timer for true output"
+    annotation (Placement(transformation(extent={{20,-70},{40,-50}})));
+
+  inner Modelica.StateGraph.StateGraphRoot stateGraphRoot
```

Comment:
It is protected and an elementary CDL block, hence other tools just need to implement the same behavior, but there is no need to parse stateGraph when exporting CDL.